zombies.js

```diff
@@ -78,8 +101,107 @@
  * @property {method} getPack              Returns private variable `pack`.
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
-
-
+  class Player {
+    constructor(name, health, strength, speed){
+      this._name = name;
+      this._health = health;
+      this._strength = strength;
+      this._speed = speed;
+      this._pack = [];
+      this._maxHealth = health;
+      this._isAlive = true;
+      this._equipped = false;
+    }
+    get name(){
+      return this._name;
+    }
+    get health(){
+      return this._health;
+    }
+    set health(newHealth){
+      this._health = newHealth;
+    }
+    get strength(){
+      return this._strength;
+    }
+    get speed(){
+      return this._speed;
+    }
+    get isAlive(){
+      return this._isAlive;
+    }
+    get equipped(){
+      return this._equipped;
+    }
+    getPack(){
+      return this._pack;
+    }
+    getMaxHealth(){
+      return this._maxHealth;
+    }
+    takeItem(item){
+      if (this.getPack().length < 3){
+        this.getPack().push(item);
+        return true;
+      } else {
+        console.log('not enough room');
+        return false;
+      }
+    }
+    discardItem(item){
+      if (this.getPack().indexOf(item) !== -1){
+        this.getPack().splice(this.getPack().indexOf(item), 1);
+        console.log(this._name + ' discarded ' + item);
+        return true;
+      } else {
+        return false;
+      }
+    }
+    checkPack(){
+      console.log(this.getPack());
+    }
+    equip(itemToEquip){
+      if (itemToEquip instanceof Weapon){
+        if (this.getPack().indexOf(itemToEquip) !== -1){
```

Comment:
Merge the top two if statements into one if statement. It might help to precalculate the values into variables first.